http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -137,13 +137,13 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         .run(authContext.copy(namespace = config.aadPodIdentityConfig.namespace))
 
       // Create relay hybrid connection pool
-      hcName = RelayHybridConnectionName(params.appName.value)
+      hcName = RelayHybridConnectionName(s"${params.appName.value}-${params.workspaceId.value}")
```

Comment:
One question from Qi on Jordan's old PR was if this was going to be too long for an HC name, could you ask in the dsp-azure channel?

Reply:
oh yeah I looked this up and the max limit is 50 chars, which I don't think will be a problem but I can add in a restriction: [Azure Resource Name Rules](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#:~:text=namespaces%20/%20HybridConnections/authorizationRules,end%20with%20alphanumeric.)